src/pages/Dashboard.vue

```diff
@@ -2,153 +2,83 @@
         farmedList(
           :expanded="expanded"
-          :farmedTotalEarned="farmedTotalEarned"
           @expand="expand"
         )
-  div(v-else)
-    .flex
-      .absolute-center
-        .row.justify-center
-          q-spinner-orbit(color="grey" size="120px")
-        h4 {{ $t('dashboard.loadingMsg')}}
 </template>
 
 <script lang="ts">
-import { defineComponent } from "vue"
+import { defineComponent, watch } from "vue"
 import { Notify } from "quasar"
-import { globalState as global } from "../lib/global"
-import * as util from "../lib/util"
+
 import farmedList from "../components/farmedList.vue"
 import netCard from "../components/netCard.vue"
 import plotCard from "../components/plotCard.vue"
-import { emptyClientData, ClientData, FarmedBlock } from "../lib/types"
-import { appConfig } from "../lib/appConfig"
+import { FarmedBlock } from "../lib/types"
+import { useStore } from '../stores/store';
+import * as util from '../lib/util';
+import * as blockStorage from '../lib/blockStorage';
 
 export default defineComponent({
   components: { farmedList, netCard, plotCard },
+  setup() {
+    const store = useStore();
+    return { store };
+  },
   data() {
     return {
-      network: {
-        state: "starting",
-        message: this.$t('dashboard.initializing'),
-        peers: 0
-      },
-      plot: {
-        state: "starting",
-        message: this.$t('dashboard.initializing'),
-        plotSizeGB: 0
-      },
-      global: global.data,
-      globalState: {
-        state: "starting",
-        message: this.$t('dashboard.initializing')
-      },
       expanded: false,
       util,
-      loading: true,
-      unsubscribe: () => null,
       peerInterval: 0,
-      clientData: <ClientData>emptyClientData
-    }
-  },
-  computed: {
-    farmedTotalEarned(): number {
-      if (!this.$client) return 0
-      return this.$client.data.farming.farmed.reduce((agg: number, val: { blockReward: number, feeReward: number }) => {
-        return val.blockReward + val.feeReward + agg
-      }, 0)
     }
   },
   async mounted() {
-    const config = await appConfig.read()
-    this.plot.plotSizeGB = config.plot.sizeGB
+    // watch for farmed blocks
+    watch(
+      () => this.store.farmedBlocks.length,
+      () => {
+        if (this.store.farmedBlocks.length) {
+          this.farmBlock(this.store.farmedBlocks[0]);
+        }
+      },
+    )
 
-    if (this.$client.isFirstLoad() === false) {
-      util.infoLogger("DASHBOARD | starting node")
-      if (config.nodeName !== "") {
-        await this.$client.startNode(config.plot.location, config.nodeName)
-      } else {
-        util.errorLogger("DASHBOARD | node name was empty when tried to start node")
-      }
-      util.infoLogger("DASHBOARD | starting farmer")
-      const farmerStarted = await this.$client.startFarming(config.plot.location, config.plot.sizeGB)
-      if (!farmerStarted) {
-        util.errorLogger("DASHBOARD | Farmer start error!")
-      }
```

Comment:
why the error handling is abandoned?

Reply:
error logging is still in place - it was moved to the store, error handling will be implemented within #253